website/docs/docs/use-dbt-semantic-layer/sl-faqs.md

```diff
@@ -66,6 +66,13 @@ The dbt Semantic Layer measures usage in distinct 'Queried Metrics'. Refer to th
 
 ## Availability
 
+<expandable alt_header="What data platforms are supported by the dbt Semantic Layer?">
+
+The dbt Semantic Layer supports the following data platforms: Snowflake, BigQuery, Databricks, or Redshift.
```

Comment:
I think we would normally do something like this as a bulleted list, yeah?
```suggestion
The dbt Semantic Layer supports the following data platforms: 

- Snowflake 
- BigQuery 
- Databricks  
- Redshift
```

Reply:
yea its not linking or steps. no strong feelings so pushing it forward and merging the pr 🙌